CONTRIBUTING.md

```diff
@@ -50,7 +50,7 @@ Please follow this style to make openshift-preflight easy to review, maintain an
 
 ### Commit Message Format
 
-We follow a rough convention for commit messages that is designed to answer two
+We follow a rough convention for committing messages that are designed to answer two
```

Comment:
"commit messages" is the correct verbage. "is" is also correct here. If you take out "for commit messages", the sentence reads "We follow a rough convention that is designed..."

Reply:
@bcrochet Corrected to original statement